src/sentry/migrations/0457_backfill_status_for_organization_mapping.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 2.2.28 on 2023-05-22 18:27
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_status_for_organizationmember(apps, schema_editor):
+    Organization = apps.get_model("sentry", "Organization")
+    OrganizationMapping = apps.get_model("sentry", "OrganizationMapping")
+    for org_mapping in RangeQuerySetWrapperWithProgressBar(
```

Comment:
This needs to backfill the _entire_ organizationmapping table. It should iterate through all organizations and create or update corresponding organizationmapping records.

Reply:
> Do you know of any place where we're instantiating a new OrganizationMapping model from an existing organization in our production code?

https://github.com/getsentry/sentry/blob/413e647dcae5c3e84534c14f31d361f792c0ce63/src/sentry/services/hybrid_cloud/organization_mapping/impl.py#L29

> How would this backfill be applied in the distributed world when we're migrating individual regional sites?

It never will be. It will be done once before this table is pulled into the control silo and never run again.